frontend/webpack.config.js

```diff
@@ -60,6 +61,11 @@ module.exports = (env, argv) => ({
       filename: 'main-[contenthash].css',
       chunkFilename: '[chunkhash].css',
     }),
+    new webpack.DefinePlugin({
+      'process.env': {
+        'ENABLE_EVIDENCE_EXPORT': process.env.ENABLE_EVIDENCE_EXPORT == 'true',
```

Comment:
one thought that I just had is that I could include this global flag on the backend and incorporate it into `UserCanExportData`, since most of the global flags are on the backend/makes more intuitive sense. I'll do that tomorrow morning